OpenSim/Common/Component.cpp

```diff
@@ -1552,51 +1552,22 @@ void Component::printSubcomponentInfo() const {
 void Component::printOutputInfo(const bool includeDescendants) const {
     using ValueType = std::pair<std::string, SimTK::ClonePtr<AbstractOutput>>;
 
-    static const std::unordered_map<std::string, std::string>
-        typeAliases{{"SimTK::Vec<2,double,1>", "Vec2"},
-                    {"SimTK::Vec<3,double,1>", "Vec3"},
-                    {"SimTK::Vec<4,double,1>", "Vec4"},
-                    {"SimTK::Vec<5,double,1>", "Vec5"},
-                    {"SimTK::Vec<6,double,1>", "Vec6"},
-                    {"SimTK::Vec<2,SimTK::Vec<3,double,1>,1>", "SpatialVec"},
-                    {"SimTK::Transform_<double>", "Transform"},
-                    {"SimTK::Vector_<double>", "Vector"}};
-    
     // Do not display header for Components with no outputs.
     if (getNumOutputs() > 0) {
         const std::string msg = "Outputs from " + getAbsolutePathName() +
             " [" + getConcreteClassName() + "]";
         std::cout << msg << "\n" << std::string(msg.size(), '=') << std::endl;
 
         const auto& outputs = getOutputs();
-        unsigned maxlen{};
-        bool printingAlias{false};
-        for(const auto& output : outputs) {
-            const auto& name = output.second->getTypeName();
-            unsigned len = name.length();
-            if(typeAliases.find(name) != typeAliases.end()) {
-                len += typeAliases.at(name).length();
-                printingAlias = true;
-            }
-
-            maxlen = std::max(maxlen, len);
-        }
+        size_t maxlen{};
+        for(const auto& output : outputs)
+            maxlen = std::max(maxlen, output.second->getTypeName().length());
```

Comment:
What happened to the alias-printing stuff that was deleted here? The description of the PR doesn't indicate that you've changed any functionality, so was the `if(typeAliases.find(name) != typeAliases.end())` always `false` for some reason?